### PR TITLE
[DRAFT] Support SVDF Op

### PR DESCRIPTION
**Pull request by seanshpark**
on-going draft to support SVDF Op in compiler frontend.

Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #8203 to verify import and missing codes of _luci_ and others.

File: compiler/luci-interpreter/src/loader/GraphLoader.cpp [hunk 1]
```diff
@@ -87,6 +87,7 @@ bool isExecutableNode(const luci::CircleNode *node)
     case luci::CircleOpcode::CIRCLESPLITOUT:
     case luci::CircleOpcode::CIRCLESPLITVOUT:
     case luci::CircleOpcode::CIRCLEUNPACKOUT:
+    case luci::CircleOpcode::CIRCLEVARIABLE:
     case luci::CircleOpcode::CIRCLEWHILEOUT:
```
Comment (BalyshevArtem) [inline compiler/luci-interpreter/src/loader/GraphLoader.cpp:91]: Could you please add your changes from this file, after that I will add changes to luci-interpreter

Comment (seanshpark) [reply compiler/luci-interpreter/src/loader/GraphLoader.cpp:91]: #8329 posted

**Comment by seanshpark** [comment]
done

**Pull request closed without merging**